client/src/modules/employees/registration/employees.html

```diff
@@ -0,0 +1,513 @@
+                      </div>
+                    </div>
+                  </div>
+
+                  <!-- Employee email -->
+                  <div
+                    class="form-group"
+                    ng-class="{'has-error' : DetailForm.email.$invalid && EmployeeRegistrationForm.$submitted}">
+
+                    <label class="col-md-3 control-label">{{ "FORM.LABELS.EMAIL" | translate }}</label>
+                    <div class="col-md-9">
+                      <input
+                        type="email"
+                        class="form-control"
+                        name="email"
+                        ng-model="EmployeeCtrl.employee.email"
+                        autocomplete="off">
+                      <div class="help-block" ng-messages="DetailForm.email.$error" ng-show="EmployeeRegistrationForm.$submitted">
+                        <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
+                      </div>
+                    </div>
+                  </div>
+
+                  <!-- Employee address -->
+                  <div
+                    class="form-group"
+                    ng-class="{'has-error' : DetailForm.address.$invalid && EmployeeRegistrationForm.$submitted}">
+
+                    <label class="col-md-3 control-label">{{ "FORM.LABELS.ADDRESS" | translate }}</label>
+                    <div class="col-md-9">
+                      <input
+                        class="form-control"
+                        name="address"
+                        ng-model="EmployeeCtrl.employee.adresse"
+                        autocomplete="off">
+                      <div class="help-block" ng-messages="DetailForm.address.$error" ng-show="EmployeeRegistrationForm.$submitted">
+                        <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
+                      </div>
+                    </div>
+                  </div>
+
+                  <!-- Employee daily salary -->
+                  <div
+                    class="form-group"
+                    ng-class="{'has-error' : DetailForm.daily_salary.$invalid && EmployeeRegistrationForm.$submitted}">
+
+                    <label class="col-md-3 control-label">{{ "FORM.LABELS.DAILY_SALARY" | translate }}</label>
+                    <div class="col-md-9">
+                      <input
+                      type="number"
+                        class="form-control"
+                        name="daily_salary"
+                        ng-model="EmployeeCtrl.employee.daily_salary"
+                        autocomplete="off">
+                      <div class="help-block" ng-messages="DetailForm.daily_salary.$error" ng-show="EmployeeRegistrationForm.$submitted">
+                        <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
+                      </div>
+                    </div>
+                  </div>
+
+                  <!-- Employee bank -->
```

Comment:
Bank is the name of the bank which is hosting the employee bank account, the bank account his a string of characters representing the employee bank account.